workspaces/default/themes/base/layouts/_app.html

```diff
@@ -2,9 +2,11 @@
 
 {-main-}
   {(partials/header.html)}
-  <div class="page">
-    {* blocks.content *}
-  </div>
+  <main>
+    <div class="page">
```

Comment:
Does it cause any issues if we just do `<main class="page">` to simplify the structure?